server/jetstream_cluster.go

```diff
@@ -4665,9 +4665,9 @@ func (js *jetStream) clusterInfo(rg *raftGroup) *ClusterInfo {
 
 	for _, rp := range peers {
 		if rp.ID != id && rg.isMember(rp.ID) {
-			lastSeen := now.Sub(rp.Last)
-			if lastSeen < 0 {
-				lastSeen = 1
+			lastSeen := time.Microsecond // Just sane small non-zero default.
```

Comment:
so when not seen at all this will now give some microseconds? Probably ok just chekcing